ambry-store/src/main/java/com.github.ambry.store/HardDeleter.java

```diff
@@ -106,7 +106,7 @@
     this.factory = factory;
     this.diskIOScheduler = diskIOScheduler;
     this.time = time;
-    scanSizeInBytes = Math.min(config.storeCleanupOperationsBytesPerSec * 10, 1024 * 1024);
```

Comment:
Throttlers in DiskIOScheduler are split to two now, one is for compaction and the other is for hardDelete. Since the name no longer need to be shared, I created `BlobStoreCompactor.COMPACTION_OPS_JOB_NAME` and `HardDeleter.HARD_DELETE_OPS_JOB_NAME`, just like `BlobStoreCompactor.INDEX_SEGMENT_READ_JOB_NAME`.